### ISSUE TITLE
Some content on manager pages disappeared. 

### ISSUE THREAD

**Issue by sean69** [modxbot/migrate#5733]
sean69 created Redmine issue ID 5733

I am trying to update an installation of revo - well it is update 2.1.3, the update was in hopes of fixing some missing manager pages:
- the package manager page only shows 2 packages while I know there are at least 10
- the system settings, lexicon management and access controls are all missing the right hand content - I see the resource tree & top navigation, just the content is missing [i.e. I can't view/update any of this!!] 

If it were permissions I would expect to see the access denied error on the manager page in question, I do not get one.

I've checked the file permissions [core/ cache, components, packages etc., they all look file. 
No errors in the modx or webserver logs ... 

I'm not sure what has happened here, nothing has been done to it since it was setup & all that stuff was working at that point. 

HELP!!

-sean

PS> I'm a modx professional partner if that counts for anything ;)
[BigBlock Studios]

**Comment by splittingred** [comment]
splittingred submitted:
- Do you have mod_security running? 
- Or any kind of opcode caching? 
- Are your file/folder permissions set right?
- Are you running out of disk space on the server?

**Comment by sean69** [comment]
sean69 submitted:
- At the time, mod_security was not installed or running [ it has been installed since though...]
- apc is setup on the server, but is configured only on a few select sites, this is not one of them [so no - no extra caching]
- I went over the permissions extensively [one of the first things] & even compared to other working installations, everything looked correct. 
- approx 30% disk space was still available. 

I never did resolve what the problem was, but I did fix it by downloading all the missing packages [by looking to see what should have been installed in the core/packages dir]  updating modx to the latest, then installing all the downloaded packages.
Sorry I don't have more info - it was kind of urgent that I fix it & had to go the ugly reinstall route.. 

-sean
